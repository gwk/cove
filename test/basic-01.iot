{
  'cmd': '$PROJ/cove.py test/basic.py 01',
}
