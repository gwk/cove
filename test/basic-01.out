cove report:

__main__: basic.py: 16 lines; 9 traceable; 0 ignored.
